lib/install/action/fetch.js

```diff
@@ -24,6 +24,6 @@ module.exports = function (top, buildpath, pkg, log, next) {
       name = pkg.package._requested.raw
   }
   log.silly('fetch', packageId(pkg))
-  cache.add(name, version, top, false, next)
+  cache.add(name, version, pkg.parent.path, false, next)
```

Comment:
This is actually in commented out code. (`action/fetch.js` is at this time, and until some registry 3 features, a no-op). It does strictly speaking fix a bug in that comment out code…

and yes, keeping this commented out action is weird, although it should be handy soon, hopefully!